CHANGELOG.next.asciidoc

```diff
@@ -146,6 +146,7 @@ https://github.com/elastic/beats/compare/v6.6.0...6.x[Check the HEAD diff]
 - Add system module. {pull}9546[9546]
 - System module `process` dataset: Add user information to processes. {pull}9963[9963]
 - Add system `package` dataset. {pull}10225[10225]
+- New iptables module that receives iptables/ip6tables logs over syslog or file. Supports Ubiquiti Firewall extensions. {issue}8781[8781] {pull}10176[10176]
```

Comment:
Same comment as in master, shouldn't this come under the Filebeat heading?

Reply:
Good catch Tudor